src/dotnet/Fable.Compiler/Transforms/Replacements.fs

```diff
@@ -1218,7 +1218,7 @@ let strings (com: ICompiler) (ctx: Context) r t (i: CallInfo) (thisArg: Expr opt
             addWarning com ctx.InlinePath r "String.Contains: second argument is ignored"
         let left = Helper.InstanceCall(c, "indexOf", Number Int32, [arg])
         makeEqOp r left (makeIntConst 0) BinaryGreaterOrEqual |> Some
-    | ("StartsWith" | "StartsWithOrdinal"), Some c, [_str] ->
+    | "StartsWith", Some c, [_str] ->
```

Comment:
Don't we need this anymore? It was added in #1566.

Reply:
@alfonsogarciacaro I know, it was added by mistake, I though it was missing, but it was actually caused by #1575